libs/platform/src/lib/components/button/button.component.ts

```diff
@@ -16,14 +16,14 @@ export type ButtonType =
     templateUrl: './button.component.html',
     styleUrls: ['./button.component.scss']
 })
-export class ButtonComponent extends BaseComponent {
-    /** Option to make to button compact. */
-    @Input()
-    compact: boolean;
```

Comment:
since you are removing this from the API the PR should start with `feat:`

Reply:
@sKudum As deno say, already existing code which is out there and we do this kind of chaing its always breaking change.

Reply:
Updated